.github/workflows/dev-mode-layer-publish.yml

```diff
@@ -0,0 +1,50 @@
+name: "Dev Mode Extension: Publish"
+on:
+  push:
+    tags:
+      - "go/sls-external-extension@[0-9]+.[0-9]+.[0-9]+"
+defaults:
+  run:
+    working-directory: go/packages/dev-mode
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 16
+          registry-url: "https://npm.pkg.github.com"
+      - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '>=1.18.0'
+      - name: Build layer
+        run: make build
+      - name: Publish layers (dev)
+        env:
+          AWS_ACCESS_KEY_ID: ${{ secrets.OPEN_DEV_AWS_ACCESS_KEY_ID }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.OPEN_DEV_AWS_SECRET_ACCESS_KEY }}
+        run: |
+          TEMP_ARRAY=($(echo $GITHUB_REF | tr "@" "\n"))
+          VERSION=${TEMP_ARRAY[@]: -1}
+          ./../../../node/scripts/publish-extension-layers.js \
```

Comment:
Dependencies need to be installed for this script to run

Reply:
I added main project dependency installs. Let me know if I should add any others 👍